Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.mm

```diff
@@ -238,4 +243,28 @@
     return [description stringByAppendingString:layerDescription];
 }
 
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+void RemoteLayerTreeNode::setAcceleratedEffectsAndBaseValues(const WebCore::AcceleratedEffects& effects, const WebCore::AcceleratedEffectValues& baseValues, RemoteLayerTreeHost* host)
+{
+    ASSERT(isUIThread());
+    ASSERT(host);
+
+    if (m_effects)
+        host->animationEffectStackWasRemoved(*m_effects);
```

Comment:
Should we compare old and new effects before calling this?

Reply:
This will _only_ be called if effects were sent over to the UI process through IPC which resulted from effects having changed.